android/app/src/main/java/org/openbot/NetworkActivity.java

```diff
@@ -193,81 +193,78 @@ protected void processImage() {
+                  new LinkedList<Detector.Recognition>();
+
+              for (final Detector.Recognition result : results) {
+                final RectF location = result.getLocation();
+                if (location != null && result.getConfidence() >= minimumConfidence) {
+                  canvas1.drawRect(location, paint);
+                  cropToFrameTransform.mapRect(location);
+                  result.setLocation(location);
+                  mappedRecognitions.add(result);
                 }
-
-                tracker.trackResults(mappedRecognitions, currFrameNum);
-                vehicleControl = tracker.updateTarget();
-                trackingOverlay.postInvalidate();
-              } else if (autoPilot != null) {
-                LOGGER.i("Running autopilot on image " + currFrameNum);
-                final long startTime = SystemClock.uptimeMillis();
-                vehicleControl = autoPilot.recognizeImage(croppedBitmap, vehicleIndicator);
-                lastProcessingTimeMs = SystemClock.uptimeMillis() - startTime;
               }
 
-              // In case control was removed from network during inference
-              if (!driveByNetwork) {
-                vehicleControl = new ControlSignal(0, 0);
-              }
+              tracker.trackResults(mappedRecognitions, currFrameNum);
+              vehicleControl = tracker.updateTarget();
+              trackingOverlay.postInvalidate();
+            } else if (autoPilot != null) {
+              LOGGER.i("Running autopilot on image " + currFrameNum);
+              final long startTime = SystemClock.uptimeMillis();
+              vehicleControl = autoPilot.recognizeImage(croppedBitmap, vehicleIndicator);
+              lastProcessingTimeMs = SystemClock.uptimeMillis() - startTime;
+            }
 
-              if (getLoggingEnabled()) {
-                sendInferenceTimeToSensorService(currFrameNum, lastProcessingTimeMs);
-              }
+            // In case control was removed from network during inference
+            if (!driveByNetwork) {
+              vehicleControl = new ControlSignal(0, 0);
+            }
 
-              computingNetwork = false;
+            if (getLoggingEnabled()) {
+              sendInferenceTimeToSensorService(currFrameNum, lastProcessingTimeMs);
+            }
 
-              updateVehicleState();
+            computingNetwork = false;
 
-              runOnUiThread(
-                  new Runnable() {
-                    @Override
-                    public void run() {
-                      // showFrameInfo(previewWidth + "x" + previewHeight);
-                      // showCropInfo(croppedBitmap.getWidth() + "x" + croppedBitmap.getHeight());
-                      showInference(lastProcessingTimeMs + "ms");
-                    }
-                  });
-            }
+            updateVehicleState();
+
+            runOnUiThread(
+                new Runnable() {
```

Comment:
Why use lambda previously but not here?

Reply:
There's no global converter in android studio to convert all these anonymous functions to lambdas at once. 
So it's more of an iterative process, where I keep replacing them as and when I find them.

Reply:
I've converted this and pushed another commit